components/zenler/package.json

```diff
@@ -0,0 +1,19 @@
+{
+  "name": "@pipedream/zenler",
+  "version": "0.0.1",
+  "description": "Pipedream Zenler Components",
+  "main": "zenler.app.mjs",
+  "keywords": [
+		"pipedream",
```

Comment:
Looks like there is an extra tab here

Reply:
Actually there is no extra tab as far as I can tell